src/transformers/utils/import_utils.py

```diff
@@ -294,6 +294,18 @@ def is_torch_available():
     return _torch_available
 
 
+def is_torch_deterministic():
+    """
+    Check whether `torch.use_deterministic_algorithms()` is set to True or torch.set_deterministic_debug_mode() is set to 1 or 2
+    """
+    import torch
+
+    if torch.get_deterministic_debug_mode() == 0:
+        return False
+    else:
+        return True
```

Comment:
not corresponds to the docstring: `torch.use_deterministic_algorithms` is not used in the code, but mentioned in the docstring.

Reply:
Should I change to "Check whether pytorch uses deterministic algorithms or torch.set_deterministic_debug_mode() is set to 1 or 2"?

Reply:
Oh `torch.use_deterministic_algorithms()` is a set function not get function.

> Check whether pytorch uses deterministic algorithms by looking if torch.set_deterministic_debug_mode() is set to 1 or 2"

how about this?